src/containers/AccountConnection.jsx

```diff
@@ -262,6 +263,12 @@ class AccountConnection extends Component {
               date={account && account.lastSync}
             /> }
 
+            {!success && account && account.auth && <KonnectorFolder
+              connector={connector}
+              account={account}
+              driveUrl={this.store.driveUrl}
```

Comment:
I agree that the fetch drive url code misses some error handling code and I will fix it. Again, I understand the point of keeping this data into the component itself, but this causes some content shift problem due to the asynchronous fetching of it.